packages/web-component/wordpress-plugin/wp-read-along-web-app-loader/README.txt

```diff
@@ -0,0 +1,42 @@
+=== Plugin Name ===
+Contributors: 
+Donate link: 
+Tags: read-along
+Requires at least: 4.7
+Tested up to: 5.4
+Stable tag: 4.3
+Requires PHP: 7.0
+License: MIT
+License URI: https://github.com/ReadAlongs/Web-Component/blob/main/LICENSE
+
+This WordPress plugin allows the deployment and activation of Read-Along generated using https://github.com/ReadAlongs/ on WordPress sites.
+== Description ==
+
+This WordPress plugin allows the deployment and activation of Read-Along generated using https://github.com/ReadAlongs/ on WordPress sites. Just wrap your <read-along> in a WordPress shortcode [wp_read_along_web_app_loader]. The plugin does not load the script on all pages on your site; it will only add the script to the pages with the shortcode.
+The shortcode accepts two optional attributes:
+ - The `src` attribute for users with custom scripts defaults to unpkg.com
+ - The `version` attribute for which version on unpkg.com defaults to the latest
```

Comment:
put semi colon before "default" on both these lines, it's confusing as is.

Reply:
DONE. Thanks